charts/rapid-response/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Change Log
 
+## v0.1.1
+
+### Bugfix
+
+* Fixed an issue that was causing k8s secrets to get generated with uppercase characters
```

Comment:
this is interesting

Reply:
Yeah the helm-controller in particular will complain about it:
```
helm-controller  Helm upgrade failed: failed to create resource: Secret "sysdig-deploy-rapidResponse-passphrase" is invalid: metadata.name: Invalid value: "sysdig-deploy-rapidResponse-passphrase": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters
```

Reply:
I see, thank you!